modules/evm/src/lib.rs

```diff
@@ -117,26 +127,54 @@ pub trait Trait: frame_system::Trait + pallet_timestamp::Trait {
 pub const STORAGE_SIZE: u32 = 64;
 
 #[derive(Clone, Eq, PartialEq, RuntimeDebug, Encode, Decode)]
-pub struct ContractInfo {
+pub struct ContractInfo<T: Trait> {
 	pub storage_count: u32,
 	pub code_hash: H256,
+	pub existential_deposit: BalanceOf<T>,
+	pub maintainer: T::AccountId,
```

Comment:
will make this `H160` simplify the code? so we don't need to to_evm_address every time?